/*
 * This file was automatically generated by EvoSuite
 * Sat Apr 22 09:51:14 KST 2017
 */

package com.sleepycat.je;

import static org.evosuite.runtime.EvoAssertions.assertThrownBy;
import static org.junit.Assert.fail;

import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.Test;
import org.junit.runner.RunWith;

@RunWith(EvoRunner.class) @EvoRunnerParameters(useVNET = true) 
public class Sequence_ESTest5 extends Sequence_ESTest_scaffolding5 {

  @Test
  public void test0()  throws Throwable  {
      DatabaseEntry databaseEntry0 = new DatabaseEntry();
      Sequence sequence0 = null;
      try {
        sequence0 = new Sequence((Database) null, (Transaction) null, databaseEntry0, (SequenceConfig) null);
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         assertThrownBy("com.sleepycat.je.Sequence", e);
      }
  }
}
